### PR TITLE
Update template.html.md

### PR DESCRIPTION
**Pull request by sprutner**
this was super confusing to me, and didn't show off the cool functionality. found out through trial and error, was surprised it worked without needing to restart...could have saved me some time.

### CODE REVIEW

File: website/source/docs/job-specification/template.html.md [hunk 1]
```diff
@@ -48,9 +48,10 @@ README][ct].
 ## `template` Parameters
 
 - `change_mode` `(string: "restart")` - Specifies the behavior Nomad should take
-  if the rendered template changes. The possible values are:
+  if the rendered template changes. Nomad will write to the filesystem in all cases.
```
Comment (dadgar) [inline website/source/docs/job-specification/template.html.md]: How about:

> Specifies the behavior Nomad should take if the rendered template changes. Nomad will always write the new contents of the template to the specified `destination`. The possible values below describe Nomad's action after writing the template to disk.

And then remove the change to `noop`.

**Comment by dadgar** [comment]
Thanks @sprutner! Glad you are liking the feature! Let me know what you think about the readability of what I suggested!

**Comment by sprutner** [comment]
Readability is great 💯

**Comment by dadgar** [comment]
@sprutner Sweet! Once the PR gets updated to that new verbiage I will merge!

**Comment by dadgar** [comment]
Closing 9a4f8a6d

**Pull request closed without merging**